docs/source/backends/vxlan_over_wireguard.rst

```diff
@@ -0,0 +1,78 @@
+============================
+VXLAN over WireGuard Backend
+============================
+
+.. include:: ../_github.rst
+
+The ``VXLAN over WireGuard`` backend extends :doc:`Wireguard backend </backends/wireguard>`
+to add configurations required for VXLAN.
```

Comment:
I think something like `configurations required for configuring VXLAN tunnels encapsulated in WireGuard tunnels.` may be more specific and clear.